.github/scripts/unittest.sh

```diff
@@ -4,15 +4,11 @@ set -euo pipefail
 
 ./.github/scripts/setup-env.sh
 
-# Prepare conda
-CONDA_PATH=$(which conda)
```

Comment:
Shorten this a little. No functional difference, but can be easier copy-pasted during SSH debug sessions in CI.